CHANGELOG.md

```diff
@@ -1,16 +1,31 @@
 # Mattermost Changelog
 
-## UNDER DEVELOPMENT Release v1.3.0
+## Release v1.3.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost changelog appears in the product's `master` branch to note key changes committed to master and are on their way to the next stable release. When a stable release is pushed the "UNDER DEVELOPMENT" heading is removed from the final changelog of the release. 
+Release date: 2015-12-16
 
-- **Release candidate anticipated:** 2015-12-10
-- **Final release anticipated:** 2015-12-16
+### Release Highlights
+
+#### iOS App
+
+- New [Mattermost iOS App](https://github.com/mattermost/ios) now available for iPhone, iPad, and iPod Touch
+- New [Mattermost Push Notification Service](https://github.com/mattermost/push-proxy) encrypts notifications before sending them to your iOS devices
```

Comment:
@it33 Same thing.  The push notification service isn't part of the 1.3 bits.  Although 1.3 pushes messages to it.